modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -109,6 +109,49 @@ TEST(Test_Caffe, read_googlenet)
     ASSERT_FALSE(net.empty());
 }
 
+TEST_P(Test_Caffe_nets, Axpy)
+{
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE)
+        throw SkipTestException("");
```

Comment:
Can we enable IE backend? Initializing `ScaleShift` layer with variable inputs by one of IE layers?